src/App.tsx

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [total] = useState(42);
```

Comment:
Why do you need a useState for this value? 
BTW you can move this variable outside the component at all
```suggestion
const TOTAL_PAGES = 42;

export const App: React.FC = () => {
```